packaging/linux/build_binaries.sh

```diff
@@ -44,9 +43,7 @@ if [ "$mode" != "production" ] ; then
   # The non-production build number is everything in the version after the hyphen.
   build_number="$(echo -n "$version" | sed 's/.*-//')"
   ldflags_client="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
-  commit_short_kbfs="$(git -C "$kbfs_repo" rev-parse --short HEAD)"
-  build_number_kbfs="$(echo -n "$build_number" | sed 's/+..*/+/')$commit_short_kbfs"
-  ldflags_kbfs="-X github.com/keybase/client/go/kbfs/libkbfs.PrereleaseBuild=$build_number_kbfs"
+  ldflags_kbfs="-X github.com/keybase/client/go/kbfs/libkbfs.PrereleaseBuild=$build_number"
```

Comment:
actually i'm not really sure what these are for. maybe useful for inspecting binaries?